kubernetes/ama-logs.yaml

```diff
@@ -967,6 +967,7 @@ spec:
               - fluentdwinaks
               - "C:\\etc\\amalogswindows\\filesystemwatcher.txt"
               - "C:\\etc\\amalogswindows\\renewcertificate.txt"
+              - MonAgentCore.exe
```

Comment:
This will fail incase of the legacy auth. This should be only checked in case of the MSI auth and livenessprobe.cpp code needs to handle this as well.